ffmpeg/ffmpeg_errors.go

```diff
@@ -49,6 +49,34 @@ func error_map() map[int]error {
 
 var ErrorMap = error_map()
 
+func non_retriable_errs() []string {
+	errs := make([]string, 0, 20)
+	// Add in Cgo LPMS specific errors
+	for _, v := range lpmsErrors {
+		errs = append(errs, v.Desc)
+	}
+	// Add in internal FFmpeg errors
+	// from https://ffmpeg.org/doxygen/trunk/error_8c_source.html#l00034
+	ffmpegErrors := []string{
+		"Decoder not found", "Demuxer not found", "Encoder not found",
+		"Muxer not found", "Option not found", "Invalid argument",
+	}
+	for _, v := range ffmpegErrors {
+		errs = append(errs, v)
+	}
+	// Add in ffmpeg.go transcoder specific errors
+	transcoderErrors := []error{
+		ErrTranscoderRes, ErrTranscoderInp, ErrTranscoderVid,
+		ErrTranscoderFmt, ErrTranscoderPrf, ErrTranscoderGOP,
+	}
+	for _, v := range transcoderErrors {
+		errs = append(errs, v.Error())
+	}
+	return errs
+}
+
+var NonRetriableErrs = non_retriable_errs()
```

Comment:
Pretty minor spelling comment:

So...it looks like "retryable" and "retriable" are both acceptable spellings at least according to this [SO post](https://english.stackexchange.com/questions/305273/retriable-or-retryable/305433) (apparently neither is in most dictionaries). But, we already use the [retryable spelling](https://github.com/livepeer/go-livepeer/search?q=retryable) in a bunch of places in go-livepeer so for consistency I recommend using that spelling.

Reply:
Ah good catch, I used them interchangeably without even noticing. Will fix!